### ISSUE TITLE
Normalization in Golub-Welsch for Gauss-Jacobi correct?

### ISSUE THREAD

**Issue by timueh** [JuliaApproximation/FastGaussQuadrature.jl#65]
First of all -- thanks for this amazing package; I've been using it quite a bit. However, I have the feeling that there is something fishy with the Golub-Welsch algorithm in `gaussjacobi()`:

For $n<=4000$ and $max(a,b)>=5$, the function `JacobiGW(n, a, b)` is called, see https://github.com/ajt60gaibb/FastGaussQuadrature.jl/blob/bd0b69e6af7dde596bc831d4beffd8c8f2d76224/src/gaussjacobi.jl#L29

In the function `JacobiGW(n, a, b)` the Golub-Welsch algorithm is called, i.e. the nodes and weights are computed based on the eigenvalues/-vectors of the Jacobi matrix. The weights are the [square of the first entry of the normalized eigenvector times the integral of the weight over the support](https://en.wikipedia.org/wiki/Gaussian_quadrature#The_Golub-Welsch_algorithm), $w_j = \mu_0 \v_1^{j}^2$. In `gaussjacobi.jl` the weights are computed here:

https://github.com/ajt60gaibb/FastGaussQuadrature.jl/blob/bd0b69e6af7dde596bc831d4beffd8c8f2d76224/src/gaussjacobi.jl#L385

with $\mu_0 = 2^(ab+1)*gamma(a+1)*gamma(b+1)/gamma(2+ab)$. But then comes the next line: 

https://github.com/ajt60gaibb/FastGaussQuadrature.jl/blob/bd0b69e6af7dde596bc831d4beffd8c8f2d76224/src/gaussjacobi.jl#L386

Why is the division necessary? In fact, this normalization leads to the weights returned by `JacobiGW()` satisfing $\sum_{i=1} w_i = 1 != \mu_0$ hence a contradiction, in general; the integral of the Jacobi weight is not one in general.

In short: shouldn't line 386 be commented out?

happy holidays! :)

**Comment by dlfivefifty** [comment]
You are correct:
```julia
julia> x,w = gaussjacobi(10,6,6)
([-0.815935, -0.66479, -0.492055, -0.302324, -0.101975, 0.101975, 0.302324, 0.492055, 0.66479, 0.815935], [0.000286535, 0.00719355, 0.05063, 0.161847, 0.280042, 0.280042, 0.161847, 0.05063, 0.00719355, 0.000286535])

julia> sum(w) # should be 0.681984681984682 from Mathematica
0.9999999999999999
```

Would you mind making a PR?

**Comment by tpapp** [comment]
Was this closed by #67?

**Comment by timueh** [comment]
Yes, it was. Thanks for the book-keeping.
